services/property-svc/internal/property/events/v1/events.go

```diff
@@ -4,27 +4,66 @@ import (
 	pb "gen/proto/services/property_svc/v1"
 	"github.com/google/uuid"
 	"hwes"
+	"property-svc/internal/property/models"
 )
 
 const (
-	PropertyCreated = "PROPERTY_CREATED_v1"
+	PropertyCreated                               = "PROPERTY_CREATED_v1"
+	PropertyDescriptionUpdated                    = "PROPERTY_DESCRIPTION_UPDATED_v1"
+	PropertySetIDUpdated                          = "PROPERTY_SET_ID_UPDATED_v1"
+	PropertyAlwaysIncludeForCurrentContextUpdated = "PROPERTY_ALWAYS_INCLUDE_FOR_CURRENT_CONTEXT_UPDATED_v1"
 )
 
 type PropertyCreatedEvent struct {
-	ID          string `json:"id"`
-	SubjectID   string `json:"subject_id"`
+	ID string `json:"id"`
+
 	SubjectType string `json:"subject_type"`
 	FieldType   string `json:"field_type"`
-	Name        string `json:"name"`
+
+	Name          string               `json:"name"`
+	FieldTypeData models.FieldTypeData `json:"field_type_data"`
```

Comment:
Description is missing

Reply:
As description is optional, we don't want to have it inside the `PropertyCreatedEvent`

In the command handler you can see that we trigger the `PropertyDescriptionUpdatedEvent` if set initially.